S100-Shop-Facade-Open-Api-Sepc-v0.1.yaml

```
@@ -0,0 +1,1145 @@
+openapi: 3.0.1
+info:
+  title: UKHO S100 Shop Facade Service APIs
+  description: |
+    The UKHO S100 Shop Facade Service APIs.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
```

If we're going to have a contact address in here, it should be valid. Suggest:

```suggestion
    email: martin.rock-evans@ukho.co.uk
```

---

Updated.